### PR TITLE
5866 - Field Filter: Field & filter icon issues

### PR DESCRIPTION
**Pull request by sesbrenosienna**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
This PR will fix a bug in the alignment of filter icons and text fields and the border radius of the dropdown field in Firefox and Safari. 

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/5866

**Steps necessary to review your pull request (required)**:
1st Issue
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/field-filter/example-short.html on Firefox
- Click on the dropdown

2nd Issue
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/field-filter/example-short.html on Safari
- See field for Datepicker/Timepicker/Lookup

3rd Issue 
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/field-filter/example-short.html on Safari
- See the dropdown field 

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [ ] A note to the change log.

### CODE REVIEW

File: src/components/datepicker/_datepicker.scss [hunk 1]
```diff
@@ -253,6 +253,7 @@ html.is-safari {
   .datepicker {
     + .trigger {
       position: relative;
+      margin-top: 2px;
```
Comment (ericangeles) [inline src/components/datepicker/_datepicker.scss:256]: It's in the main already. I believe this was part of your previous PR.

File: src/components/dropdown/_dropdown.scss [hunk 2]
```diff
@@ -1184,13 +1184,6 @@ div.dropdown-xs,
     }
   }
 
-  div.dropdown,
```
Comment (ericangeles) [inline src/components/dropdown/_dropdown.scss:1187]: Same in here

**Review by ericangeles — CHANGES_REQUESTED** [review]

**Comment by tmcconechy** [comment]
@sesbrenosienna - as same as @ericangeles i was wondering about the changes to the individual components in the css. Seems out of scope and that they should be isolated to field filter scss. Is it just that we arent updated with main?

**Review by yohannahbautista — COMMENTED** [review]
Not sure if it should be in this issue, but something to note: the icons "twitch" when you expand the dropdowns in Chrome, Safari, and Firefox (and in Firefox one of them disappears). It works fine in Edge so I'm basing it on that.

**Comment by tmcconechy** [comment]
One thing to note is the field filter isn't very commonly used and we deprecated it. So maybe leave any remaining issues as its not super important to get 100% right.

**Review by tmcconechy — APPROVED** [review]

**Review by ericangeles — APPROVED** [review]

**Pull request merged**